assets/css/main.css

```diff
@@ -14,3 +14,5 @@
 @import "sections/form";
 
 @import "pages/home";
+
+@import "tasks/_contact-tasks"
```

Comment:
In trello I saw that the task from global.css was completed. Do you know which branch this file is in? You have to add it because I'm afraid it will change the way your work looks. After that, you will have to add some tweaks to your work.